extensions/strategies/trust_distrust/strategy.js

```diff
@@ -0,0 +1,135 @@
+var z = require('zero-fill')
+  , n = require('numbro')
+
+module.exports = function container (get, set, clear) {
+  return {
+    name: 'trust_distrust',
+    description: 'Sell when price higher than $sell_min% and highest point - $sell_threshold% is reached. Buy when lowest price point + $buy_threshold% reached.',
+
+    getOptions: function () {
+      this.option('period', 'period length', String, '30m')
```

Comment:
You're missing a `min_periods` value here, which causes a NaN when trying to calculate the number of days to backfill [here](https://github.com/DeviaVir/zenbot/blob/deviavir/strategies/commands/trade.js#L76).